packages/utils/src/math/range/index.ts

```diff
@@ -0,0 +1,34 @@
+import { isNil } from '../../validator';
+
+const getRangeValue = (start: number, end: number, step: number = 1) => {
+  const result = [];
+
+  for (let value = start; value <= end; value += step) {
+    result.push(value);
+  }
+
+  return result;
+};
+
+export function range(end: number): number[];
+export function range(start: number, end: number): number[];
+export function range(start: number, end: number, step?: number): number[];
+export function range(startOrEnd: number, end?: number, step?: number) {
+  if (isNil(end)) {
+    return getRangeValue(1, startOrEnd);
+  }
```

Comment:
```suggestion
  if (isNil(end)) {
    getRangeValue(0, start);
  }
```
시작을 기본 값이 1이아닌 0이 적절한 것 같습니다! 
파이썬의 경우에도 range의 결과는  아래와 같습니다

```python
print(list(range(5))); // [0, 1, 2, 3, 4]
```